### PR TITLE
[ACNA-1107] Omit old logs while continuous logs fetching

### LINKED ISSUE #385: aio app logs --tail shows last log before running the command
**Describe the bug**
relates to https://github.com/adobe/aio-cli-plugin-app/issues/277

Similar to the above issue where this is fixed in `aio app run`, `aio app logs --tail` needs fixing

### PR DESCRIPTION
**Pull request by florind12**
## Description

`startTime` is being used to filter out the logs to be printed.
While `--tail` or any other continuously fetching param, it should only fetch logs since `Date.now()`.

## Related Issue
Fixes #385

## How Has This Been Tested?

Prerequisites: aio cli installed & a firefly project.

Steps: 
1. cd into the generated firefly project. 
2. run `aio app logs --tail`.
3. While `aio app logs --tail` is running, generate some logs via the firefly app.
Expected: Only logs following the `aio app logs --tail` should be printed.  

Steps: 
1. cd  firefly project.
2. run `aio app logs`
Expected: Logs prior to `aio app logs` should be printed. 

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by shazron** [comment]
@florind12  please fill in the section `How Has This Been Tested?`

File: src/commands/app/logs.js [hunk 1]
```diff
@@ -78,7 +78,10 @@ class Logs extends BaseCommand {
 
     try {
       const owConfig = { ow: Object.values(fullConfig.all)[0].ow }
-      await rtLib.printActionLogs(owConfig, this.log, flags.limit, filterActions, flags.strip, flags.poll || flags.tail || flags.watch)
+      const tail = flags.poll || flags.tail || flags.watch
+      const startTime = tail ? Date.now() : 0
+
+      await rtLib.printActionLogs(owConfig, this.log, flags.limit, filterActions, flags.strip, flags.poll || flags.tail || flags.watch, undefined, startTime)
```
Comment (shazron) [inline src/commands/app/logs.js:84]: line 84: `flags.poll || flags.tail || flags.watch` should be just `tail` as you defined in line 81

**Review by shazron — CHANGES_REQUESTED** [review]

File: src/commands/app/logs.js (hunk 1 above)
Comment (florind12) [reply src/commands/app/logs.js:84]: oh, I missed that. 
Thanks!

**Comment by florind12** [comment]
Closing this...
Issue addressed here: https://github.com/adobe/aio-lib-runtime/pull/67

**Pull request closed without merging**